americangut/util.py

```diff
@@ -201,9 +298,10 @@ def fetch_study(accession, metadata_path, fasta_path):
         md = fetch_metadata_xml(sample)
         all_md[qiimedb_samplename] = md
         all_cols.update(md)
-
+        break
         # write out fasta
         try:
+            raise ValueError()
```

Comment:
Since this is raising a ValueError, is the following couple of lines really necessary?  It looks like they are being short-circuited